### PR TITLE
Update to 2.0

### PR DESCRIPTION
**Pull request by nerfkidjcb**
- Updates the mod to 2.0 without changing anything else

### CODE REVIEW

File: blahaj/data.lua [hunk 1]
```diff
@@ -1,57 +1,61 @@
...
-            {
-              type = "damage",
-              damage = {
-                amount = -20,
-                type = "physical"
-              }
+    type = "use-on-self",
+    attack_parameters = {
+        type = "projectile",
+        activation_type = "consume",
```
Comment (ading2210) [inline blahaj/data.lua]: Hey don't change this. We don't want to eat the blahaj.

File: blahaj/data.lua [hunk 2]
```diff
@@ -1,57 +1,61 @@
...
+        filename = "__blahaj__/graphics/blahaj/blahaj_entity.png",
+        priority = "extra-high",
+        width = 32,
+        height = 32
+    }, { -- some mods expect there to be two fish images
+        filename = "__blahaj__/graphics/blahaj/blahaj_entity.png",
+        priority = "extra-high",
+        width = 32,
+        height = 32
+    }
```
Comment (ading2210) [inline blahaj/data.lua]: Please don't change the existing indentation for no reason.

**Review by ading2210 — CHANGES_REQUESTED** [review]

**Comment by nerfkidjcb** [comment]
Sorry I'll do better lol. One sec

File: blahaj/data.lua (hunk 1 above)
Comment (nerfkidjcb) [reply blahaj/data.lua]: Ohhh, I thought I had broken it, I didn't realise it wasnt meant to be consumable

**Comment by nerfkidjcb** [comment]
All done, apologies on being lazy. Also congrats on the chrome bug report!

**Comment by ading2210** [comment]
The mod functions properly in 2.0 which is great, but I really don't think blahaj should spoil lol

image

**Comment by ading2210** [comment]
I'm not exactly sure how to remove the spoilage mechanic here. Setting `data.raw["capsule"]["raw-fish"].spoil_result` and `data.raw["capsule"]["raw-fish"].spoil_ticks` have no effect for some reason.

**Comment by ading2210** [comment]
I figured out why the spoilage couldn't be removed before. I just needed to make the mod load after space age by setting it as an optional dependency.

**Pull request merged**